queries/useGroupMembersQuery.ts

```diff
@@ -36,18 +38,18 @@ const fetchGroupMembers = async (
 
 type IGroupMembersQueryConfig = {
   account: string;
-  group: GroupWithCodecsType | undefined | null;
+  topic: ConversationTopic;
   queryOptions?: Partial<UseQueryOptions<GroupMembersSelectData>>;
 };
 
-const groupMembersQueryConfig = (
+const getGroupMemberQueryOptions = (
   args: IGroupMembersQueryConfig
 ): UseQueryOptions<GroupMembersSelectData> => {
-  const { account, group, queryOptions } = args;
```

Comment:
transitioning away from passing objects to hooks and instead relying on primitives